includes/Story_Post_Type.php

```diff
@@ -523,6 +526,11 @@ public function filter_amp_story_element_validation_error_sanitized( $sanitized,
 			return false;
 		}
 
+		// Skip sanitization for missing vide posters.
+		if ( isset( $error['node_name'] ) && 'amp-video' === $error['node_name'] ) {
```

Comment:
As a follow-up, we should at least extend this to all `amp-*` elements.

It's ok for sanitization to remove HTML added by other plugins, but it should never remove elements added by our own editor.

Reply:
I will create a new ticket for discussing this.

Reply:
#5226